.github/tests/fork12-pessimistic.yml

```
@@ -5,7 +5,10 @@ args:
   sequencer_type: erigon
   erigon_strict_mode: false
   gas_token_enabled: false
-  zkevm_use_real_verifier: true
+  zkevm_use_real_verifier: false
   enable_normalcy: true
   agglayer_prover_sp1_key: ""
-
+  # activate the `MockProver` for `agglayer-prover` as primary executor
+  agglayer_prover_primary_prover: mock-prover
+  # activate the `NetworkProver` for `agglayer-prover` as primary executor
+  # agglayer_prover_primary_prover: network-prover
```

Maybe something simpler?

```suggestion
  agglayer_prover_primary_prover: mock-prover # use local mock prover
  # agglayer_prover_primary_prover: network-prover # use sp1 prover
```